src/app/profiles/App/Pages/Subscriptions/Subscriptions.tsx

```diff
@@ -11,25 +11,27 @@ import { TwoColumns } from 'components/atoms';
 import { SlowerMessageBox } from 'components/molecules/SidebarBox';
 import pathToContributor from 'app/profiles/App/pathToContributor';
 import SimilarProfiles from 'app/profiles/App/SimilarProfiles';
-import Filters from 'components/molecules/Filters/RadiosFilters';
-import useContributorsFilters from 'app/profiles/App/useContributorsFilters';
+import useContributorsFilters from 'app/profiles/hooks/useContributorsFilters';
 import {
   ContributorExampleLink,
   ContributorProfileListItem
 } from '../Profiles/List/ProfileList';
 import { Aside, MainCol } from '../Profiles/Profile/Profile';
 import ProfileTabs from '../../ProfileTabs';
 import { Arrow } from '../../../../../components/atoms/icons';
+import Search from '../../../../../components/molecules/Search/Search';
```

Comment:
Prefer absolute import...

Reply:
we need to setup eslint to do that automatically...

Reply:
Yeah I know, but I failed to do so... here is some background:
https://github.com/dis-moi/extension/pull/900#issue-614630571